.circleci/config.yml

```diff
@@ -198,6 +206,7 @@ workflows:
   default_workflow:
```

Comment:
The `lint` job should gate either unit_tests or integration_tests & build to fail the pipeline early when the linter fails. The lint job should also be added to the RC & Rebuild workflows (a rebase might be needed to see where lint fits into the RC workflow)

Reply:
I've added linting for the RC, Build, and Rebuild workflows. I don't think we should gate on unit tests or integration tests however. As a developer, I would like to see all unit tests and integration passing even if the linter complains. 

Preventing a build does sound great though, so I've made sure it is required on those workflows.

Reply:
lgtm!